Makefile

```diff
@@ -145,6 +145,9 @@ else
 	$(error Please set GOOS=linux for building the image)
 endif
 
+clusterctl-image: generate fmt vet manifests
+	docker build . -f cmd/clusterctl/Dockerfile -t "$(REGISTRY)/openstack-cluster-api-clusterctl:$(VERSION)"
+
 upload-images: images
```

Comment:
I think we also need to update target of `images` by adding `clusterctl-image`, and push it to docker hub when `upload-images`.

Reply:
updated.